src/test/java/com/blazemeter/jmeter/correlation/core/templates/LocalConfigurationTest.java

```diff
@@ -37,408 +45,457 @@
-        configuration.getRepositoriesNames());
-  }
-
-  @Test
-  public void shouldNotDeleteRepositoryWhenDeleteRepositoryNotAdded() {
-    configuration.removeRepository(secondRepository.getName());
-    assertEquals(Arrays.asList(CENTRAL_REPOSITORY_NAME,
-            localRepository.getName(), firstRepository.getName()),
-        configuration.getRepositoriesNames());
-  }
-
-  @Test
-  public void shouldReturnOnlyRepositoriesWithInstalledTemplateWhenGetRepositoriesWithInstalledTemplates()
-      throws ConfigurationException {
-
-    prepareExpectedRepositoryWithInstalledTemplates();
-
-    configuration.addRepository(secondRepository.getName(), secondRepositoryURL);
-    configuration
-        .manageTemplate(LocalConfiguration.INSTALL, firstRepository.getName(), firstTemplateID,
-            firstTemplateVersion);
-
-    List<CorrelationTemplatesRepositoryConfiguration> expected = Arrays
-        .asList(localRepositoryConfiguration, repositoryWithInstalledTemplates);
-
-    List<CorrelationTemplatesRepositoryConfiguration> actual = configuration
-        .getRepositoriesWithInstalledTemplates();
-
-    //We are taking the second since the first it's the default Siebel
-    assertTrue(expected.size() == actual.size() && actual.get(1).equals(expected.get(1)));
-  }
-
-  private void prepareExpectedRepositoryWithInstalledTemplates() {
-    when(repositoryWithInstalledTemplates.getName()).thenReturn(FIRST_REPO_NAME);
-    when(repositoryWithInstalledTemplates.getUrl()).thenReturn(firstRepositoryURL);
-    when(repositoryWithInstalledTemplates.getInstalledTemplates())
-        .thenReturn(new HashMap<String, String>() {{
-          put(firstTemplateID, firstTemplateVersion);
+public class LocalConfigurationTest extends WiredBaseTest {
+    private static final String VERSION_ONE = "1.0";
+    private static final String DUMMY_FILE_VERSION_ONE = "dummy-1.0.jar";
+    private static final String SIEBEL_REPO_NAME = "siebel";
+    private static final String FIRST_REPO_NAME = "first";
+    private static final String TEST_REPO_NAME = "test";
+    private static final String SECOND_REPO_NAME = "second";
+    private static final String firstRepositoryURL = "localhost/firstRepository.json";
+    private static final String secondRepositoryURL = "localhost/secondRepository.json";
+    private static final String DEFAULT_ULR = "http://localhost.com/1";
+    private static final String JAR_SUFFIX = ".jar";
+    private static final String firstTemplateID = "first";
+    private static final String firstTemplateVersion = "1.0";
+    private static final String firstTemplateDescription = "Description1";
+    private static final String REPOSITORY_ID = "R1";
+    private static final String SIEBEL_ID = "siebel";
+    private static final String SIEBEL_DESCRIPTION = "<p>This is the first version of the " + "<b>SIEBEL CRM</b> correlation recorder rules</p> " + "<p>It contains 16 of the most used Correlations for the platform.</p>";
```

Comment:
These lines are super long, is that being done on purpose? Just for consistency, maybe it is worth formatting them.

A silly comment, since it only happens in the tests, but still 😉